Libraries/Catch2Interface/TestResult.cs

```diff
@@ -96,6 +96,10 @@ public TestResult(Reporter.TestCase testcase, Settings settings, bool combined)
             Duration = _testcase.OverallResult.Duration;
             Name = _testcase.Name;
             Outcome = _testcase.OverallResult.Success ? TestOutcomes.Passed : TestOutcomes.Failed;
+            if(testcase.OverallResult.Skipped)
```

Comment:
Likely should be `if (_testcase.OverallResult.Skipped)`. Missing the underscore.